Feedback on 500 error page (#7106)
umbraco/UmbracoDocs: 15/umbraco-cms/tutorials/custom-error-page.md

    app.UseExceptionHandler("/error");
}
```

{% endcode %}

{% hint style="info" %}
To test this locally, replace `app.UseDeveloperExceptionPage();` by `app.UseExceptionHandler("/error");`. Otherwise, you will get the default error page.
{% endhint %}

### Trigger a 500 error (for testing)

To trigger a 500 error, change a `Model.Value` property in your template. For example, if your Document Type has a property called `test`you would normally render it with:

```csharp
@Model.Value("test")
```

To deliberately cause an error, change it to something invalid like:

```csharp
@Model.ValueTest("test")
```

This should throw an error, triggering your custom 500 page.

### Handling app startup failures

When Umbraco fails to start, you may see a blank screen or receive a `500.30` or `502.5` error. These indicate the web application crashed or failed to initialize.

> #### Why the app can't serve its own error page?

Reviewer: A small suggestion to update the wording of this heading.

```suggestion
#### Why can't the app serve an error page?
```
